examples/local/fivenodenetwork/main.go

```diff
@@ -66,7 +66,16 @@ func main() {
 
 func run(log logging.Logger, binaryPath string) error {
 	// Create the network
-	nw, err := local.NewDefaultNetwork(log, binaryPath)
+	config := local.NewDefaultConfig(binaryPath)
+	config.LogLevel = "ERROR"
+	for i := range config.NodeConfigs {
+		nodeName := fmt.Sprintf("node-%d", i)
+		config.NodeConfigs[i].Name = nodeName
+		config.NodeConfigs[i].ImplSpecificConfig = local.NodeConfig{
+			BinaryPath: binaryPath,
+		}
+	}
+	nw, err := local.NewNetwork(log, config)
```

Comment:
We can remove these changes -- all this is doing is changing the log level of the network runner (as opposed to any of the nodes running in the network) to `ERROR`, which I don't think we want to do